### PR TITLE
Add DKIM to docker-compose

### PR DESCRIPTION
**Pull request by kimsible**
Issue: https://github.com/Chocobozzz/PeerTube/issues/2545

Add DKIM generation keys and DKIM signature sending with the current image `mwader/postfix-relay`.

### CODE REVIEW

File: support/docker/production/.env [hunk 1]
```diff
@@ -20,6 +20,9 @@ PEERTUBE_SMTP_TLS=false
 PEERTUBE_SMTP_DISABLE_STARTTLS=false
 PEERTUBE_ADMIN_EMAIL=<MY EMAIL ADDRESS>
 POSTFIX_myhostname=<MY DOMAIN>
+# If you need to generate a list of sub/DOMAIN keys
+# pass them as a whitespace separated string <DOMAIN>=<selector>
+OPENDKIM_DOMAINS=<MY DOMAIN>=peertube
```
Comment (Chocobozzz) [inline support/docker/production/.env]: I think we should disable DKIM by default

Comment (kimsible) [reply support/docker/production/.env]: Without DKIM signature, as Postfix is the default smtp relay, DMARK verification fails so every email go into the spams, that's my initial though about enable DKIM. There has been 2 fixes on the image `mwader/postfix-relay`.

Comment (Chocobozzz) [reply support/docker/production/.env]: I'm not sure to understand: most people won't have a DMARC record. While I agree this is better to have SPF/DKIM and DMARC, we cannot force it by default.

Comment (kimsible) [reply support/docker/production/.env]: Enabling DKIM with that image simply means that DKIM signature will be sent according to the RSA key automatically generated. 

- With DKIM enabled and no DMARC record, RSA keys will be generated, mails sent but no guarantee to not be received as a spam.
- Without DKIM enabled and no DMARC record, no RSA keys, mails are sent and the same thing happens : spam.

Enable it won't change the final behaviour, sending mail even if there is no DMARC record.

I think enabling that feature is not forcing to set a DMARC record but showing a way for a good practice with Postfix as a smtp.

I also think that if Postfix is just partially setup (as a local smtp), we also should disable it by default and let in any case the user configure all the mail part alone.

**Comment by Nutomic** [comment]
FYI since I set the correct reverse dns for my instance, I didnt have any complaints about receiving mail from peertube.social. So DKIM doesnt seem to be required from my perspective.

**Pull request merged**